src/app/content/highlights/components/EditCard.spec.tsx

```diff
@@ -20,9 +20,9 @@ import * as onClickOutsideModule from './utils/onClickOutside';
 import { MAIN_CONTENT_ID } from '../../../context/constants';
 import { renderToDom } from '../../../../test/reactutils';
 
-jest.mock('./ColorPicker', () => (props: any) => <div mock-color-picker {...props} />);
-jest.mock('./Note', () => (props: any) => <div mock-note {...props} ref={props.textareaRef} />);
-jest.mock('./Confirmation', () => (props: any) => <div mock-confirmation {...props} />);
+jest.mock('./ColorPicker', () => (props: any) => <div mock-color-picker='true' data-props={props} />);
+jest.mock('./Note', () => (props: any) => <div mock-note='true' data-props={props} ref={props.textareaRef} />);
+jest.mock('./Confirmation', () => (props: any) => <div mock-confirmation='true' data-props={props} />);
```

Comment:
It makes sense that these shouldn't get spread onto the div, but I wonder if they should get added at all, they don't seem to be used?

Reply:
they're there so that they show up in the snapshot, but react was complaining about the non-divvy attributes